peekingduck/pipeline/nodes/model/posenet.py

```diff
@@ -22,12 +22,67 @@
 
 
 class Node(AbstractNode):
-    """PoseNet node class that initalises a PoseNet model to detect poses from
-    an image
+    # TODO: Add in glossary for keypoint indices
+    """PoseNet node that initalises a PoseNet model to detect human poses from
+    an image.
+
+    The PoseNet node is capable of detecting multiple human figures
+    simultaneously per inference and for each detected human figure, 17 keypoints
+    are estimated. The keypoint indices table can be found :term:`here <keypoint indices>`.
+
+    Inputs:
+        |img|
+
+    Outputs:
+        |bboxes|
+
+
+        |keypoints|
+
+
+        |keypoint_scores|
+
+
+        |keypoints_conns|
```

Comment:
Now it also returns `bbox_labels` following Diwei's recently PR

Reply:
added